autoblocks/_impl/global_state.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import threading
+from typing import Coroutine
+from typing import List
+from typing import Optional
+
+import httpx
+
+_client: Optional[httpx.AsyncClient] = None
+_loop: Optional[asyncio.AbstractEventLoop] = None
+_started: bool = False
+
+
+def init(client: httpx.AsyncClient) -> None:
```

Comment:
this shouldn't take any arguments, it should just create a client. otherwise it's not global